wazo_calld/plugins/calls/services.py

```diff
@@ -328,7 +328,7 @@ def hangup_user(self, call_id, user_uuid):
         self._verify_user(call_id, user_uuid)
         self._ari.channels.hangup(channelId=call_id)
 
-    def connect_user(self, call_id, user_uuid):
+    def connect_user(self, call_id, user_uuid, timeout=30):
```

Comment:
The default value of 30 seconds should only be defined once. It is currently defined twice: here and in the schema.